django/db/backends/oracle/compiler.py

```diff
@@ -42,7 +42,36 @@ def as_sql(self, with_limits=True, with_col_aliases=False, subquery=False):
 
 
 class SQLInsertCompiler(compiler.SQLInsertCompiler, SQLCompiler):
-    pass
+    def as_sql(self):
```

Comment:
I think it's possible. Is it preferred way? As I understand only oracle has syntax like this
```sql
INSERT ALL
 INTO <table> (fields) VALUES (values),
 ...
SELECT * FROM DUAL;
```
instead of
```sql
INSERT INTO <table> (fields)
VALUES (values), ...
```